pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
yes, and if we tighten validation, we should add a comment to `ValidateRouteStatusUpdate` noting that we should not call ValidateRouteUpdate or we'll break status updates on old invalid routes